Sources/StreamChatUI/Components.swift

```diff
@@ -426,11 +426,27 @@ public struct Components {
     // MARK: - Composer VoiceRecording components
 
     /// A flag which determines if `VoiceRecording` feature will be enabled.
-    public var voiceRecordingEnabled = false
+    public var isVoiceRecordingEnabled = false
+
+    /// When set to `true` recorded messages can be grouped together and send as part of one message.
+    public var isVoiceRecordingConfirmationRequiredEnabled = true
```

Comment:
Nit: I think it is beneficial if we explain what will happen if we set this to false since it could not be 100% clear. Since a customer's first idea could be, "How do I send the audio messages instantly?" and then they don't find any variable with this name, but they will find a comment. WDYT? 
```suggestion
    /// When set to `true`, recorded messages can be grouped together and sent as part of one message.
    /// When set to `false`, recorded messages will be sent instantly.
    public var isVoiceRecordingConfirmationRequiredEnabled = true
```

Reply:
Shall we do it in a follow up PR please?

Reply:
Yes 100% ✅